etcddb/etcddb_client.go

```diff
@@ -6,15 +6,16 @@ import (
 	"strings"
 	"time"
 
+	"github.com/spf13/viper"
 	"go.etcd.io/etcd/clientv3"
 )
 
 const (
-	// ConnectionTimeout connectio timeout
-	ConnectionTimeout = 5 * time.Second
+	// DefaultConnectionTimeout connection timeout in seconds
+	DefaultConnectionTimeout = 5
```

Comment:
I am sorry for a late note but, it probably makes sense using milliseconds for timeout here.